cmd/integration_test/build/buildspecs/cloudstack-test-eks-a-cli.yml

```diff
@@ -94,6 +96,11 @@ phases:
         if [ -f ./bin/local-bundle-release.yaml ]; then
           BUNDLES_OVERRIDE=true
         fi
+      - STAGE="dev"
+      - |
+        if [[ "$(CODEBUILD_BUILD_ID)" =~ "aws-staging-eks-a-build" ]]; then
```

Comment:
How will [[ "$(CODEBUILD_BUILD_ID)" =~ "aws-staging-eks-a-build" ]] work for E2E tests though? The job name is different there.

Reply:
This applies to all buildspecs below